.cirrus.yml

```diff
@@ -656,18 +657,9 @@ container_integration_test_task:
     # Docs: ./contrib/cirrus/CIModes.md
     only_if: *only_if_int_test
     depends_on: *build
-    matrix: &fedora_vm_axis
-        - env:
-              DISTRO_NV: ${FEDORA_NAME}
-              VM_IMAGE_NAME: ${FEDORA_CACHE_IMAGE_NAME}
-              CTR_FQIN: ${FEDORA_CONTAINER_FQIN}
-        - env:
-              DISTRO_NV: ${PRIOR_FEDORA_NAME}
-              VM_IMAGE_NAME: ${PRIOR_FEDORA_CACHE_IMAGE_NAME}
-              CTR_FQIN: ${PRIOR_FEDORA_CONTAINER_FQIN}
-              CI_DESIRED_DATABASE: boltdb
```

Comment:
Tentative LGTM but I need to think about it, gimme a little while to figure out if this (container boltdb) has ever caught a real bug. Right now I can't even think of a way to find out.

Reply:
Fair, I cannot remember one. I don't know what the original purpose of the container test is but in my eyes the important bit is that systemd is not running there so we can make sure that podman in general works without systemd booted. I don't see what the point of different db's here is or why we would need to keep testing two different fedora versions for the container task.